src/main/java/Runner.java

```diff
@@ -20,7 +21,7 @@
             new Airport("Chengdu Shuangliu International Airport", "CTU", "Shuangliu-Wuhou, Chengdu, Sichuan")
     );
 
-    static List<Object> aircrafts = Arrays.asList(
+    static List<Plane> aircrafts = Arrays.asList(
```

Comment:
maybe Aircraft is a more appropriate name since it contains also helicopter and drone.